app/helpers/helpers.rb

```diff
@@ -0,0 +1,10 @@
+class Helpers
+    def self.current_user(session)
+        @user = User.find_by(id: session[:user_id])
+    end
+
+    def self.is_logged_in?(session)
+        @user = User.find_by(id: session[:user_id])
+        !@user.nil?
+    end
+end
```

Comment:
We could just drop this helper methods into the ApplicationController, in a `helpers do ... end` block, like so:

```
# in class ApplicationController
helpers do
  def self.current_user(session)
      @user = User.find_by(id: session[:user_id])
  end

  def self.is_logged_in?(session)
     @user = User.find_by(id: session[:user_id])
     !@user.nil?
  end
end
```